benefits/working_hours.md

```diff
@@ -0,0 +1,13 @@
+# Working Hours
+
+We care about the quality of the work we produce rather than the amount of hours worked. We believe we individually know how we work best and can therefore set our own hours.
+
+Some things to note:
+
+ - We do not have a set amount of hours you need to work in a day or a week. Some people will work 6 hours, some 7, sometimes more than 8. Many of us will vary our hours day to day whilst others prefer routine.
+ - We do not have a specific start time but we do have our All Hands three times a week at 0950 so you should get into the office or dial in before that time.
```

Comment:
Would be good to know where to dial in _to_, whether it's a general channel or someone specific - tried to do this last week whilst WFH, I ultimately missed out on the meeting despite my best efforts to join.